server.go

```diff
@@ -266,6 +266,29 @@ func CustomCodec(codec Codec) ServerOption {
 	})
 }
 
+// ForcedCodec returns a ServerOption that sets a codec for message marshaling and unmarshaling.
+//
+// This will override any lookups by content-subtype for Codecs registered with RegisterCodec.
+//
+// See Content-Type on
+// https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#requests for
+// more details. Also see the documentation on RegisterCodec and
+// CallContentSubtype for more details on the interaction between encoding.Codec and
+// content-subtype.
+//
+// This function is provided for advanced users; prefer to use only
+// CallContentSubtype to select a registered codec on the client instead.
+//
+// Experimental
+//
+// Notice: This API is EXPERIMENTAL and may be changed or removed in a
+// later release.
+func ForcedCodec(codec encoding.Codec) ServerOption {
```

Comment:
I don't know how to call it, please suggest.

Reply:
Yes, the lack of namespacing here is an issue.

Let's try `ForceServerCodec()`.